packages/core/src/middlewares/model/list_all_tool.ts

```
@@ -0,0 +1,56 @@
+        }
+    })
+
+    chain
+        .middleware('list_all_tool', async (session, context) => {
+            const {
+                command,
+                options: { page, limit }
+            } = context
+
+            if (command !== 'list_tool')
+                return ChainMiddlewareRunStatus.SKIPPED
```

ESLint

[error] Delete `⏎···············`

(prettier/prettier)